app/assets/javascripts/admin/templates/web-hooks-show.hbs

```diff
@@ -0,0 +1,78 @@
+{{#link-to 'adminWebHooks' class="go-back"}}
+  {{fa-icon 'arrow-left'}}
+  {{i18n 'admin.web_hooks.go_back'}}
+{{/link-to}}
+
+<div class='web-hook-container'>
+  <p>{{i18n 'admin.web_hooks.detailed_instruction'}}</p>
+  <form class='web-hook form-horizontal'>
+    <div>
+      <label for='payload-url'>{{i18n 'admin.web_hooks.payload_url'}}</label>
+      {{text-field name="payload-url" value=model.payload_url placeholderKey="admin.web_hooks.payload_url_placeholder"}}
```

Comment:
lets add a just in time validation for the input fields that require it. `input-validation` should do that trick. Also, I noticed that we are not making sure it is a valid URL